### DISCUSSION TITLE
Some questions about year not being shown in db and how to exclude album art

### DISCUSSION THREAD

**Discussion by lxsdd** [gerbera/gerbera#2842] [Q&A] [answered]
Hi guys,

i'm trying to make the jump from minimserver to gerbera, as i really like to customize upnp tree with custom mp3 tags - which seems to be possible but a big task using scripting...

first of all i'm wondering why year isn't fetched or shown in standard db tree view for now (year folder stays black in db view - mp3s are tagged id3v2.3, standard scheme)?

also is there any way to excluded audio folders being fetched to photo db? i mean all the album cover art included as cover.jpg in every album folder doesn't need to be part of my photo db.

running gerbera 1.12.1 (https://gerbera.jfrog.io/ distribution) on debian 11 with sqlite db.

**Comment by KarlStraussberger** [comment, marked as answer]
First of all the problem with the year was a bug in builtin layout. Thanks for reporting this. In order to be more flexible you should switch to js layout.

If you want to get rid of the album covers in the Photo virtual tree you need to add  `media-type="Music|AudioBook|Movie"` th the `directory` element in `<autoscan>` section of your `config.xml`

**Reply by lxsdd** [reply]
Thanks for replying! JS-Layout is next on my todo-list. Will take some time for me to understand containers, objects, js at all, but hopefully will get there. Was always wondering why it couldn't get managed like creating and viewing tags in foobar2000, but it is what it is and most likely the nature of upnp.
